takwimu/settings.py

```diff
@@ -97,6 +97,21 @@
     'UA-44795600-8',  # HURUmap
 ]
 
+HURUMAP['primary_release_year'] = {
+    'country': 2017,
+}
+HURUMAP['latest_release_year'] = '2017'
+HURUMAP['primary_dataset_name'] = 'Census'
+HURUMAP['available_release_years'] = {
+    'tanzania': [2017],
+    'kenya': [2017],
+    'ethiopia': [2017],
+    'senegal': [2017],
+    'south africa': [2017],
+    'nigeria': [2017],
+}
+
```

Comment:
+ Isn't the structures of `available_release_years` suppose to follow `geo` levels like `primary_release_year`? i.e continent, country, etc. If not, then this is cool, otherwise I'd suggest having something like `HURUMAP['tz']['available_release_years'] = {`
 + Wouldn't using country codes be better than full names?

Reply:
@kilemensi there's also no data for country, so the only levels to be concerned with are country and the first subnational level

Reply:
> @kilemensi yes it's supposed to follow `geo` levels but I updated the function that gets the release years to search for them per country instead, I modified https://github.com/OpenUpSA/wazimap/blob/eccb51b5f28365b825e8320a49227ad81f4cfc6b/wazimap/data/utils.py#L345

Wouldn't it be better to leave the default behaviour so we don't end up diverging too much from wazimap? Is there any upside to doing it this way?

Reply:
> @kilemensi there's also no data for country, so the only levels to be concerned with are country and the first subnational level

Yes but remember the end goal of HURUmap is to go deeper than just national and level 1.

Reply:
@kilemensi I meant continent not country, it will work fine for country and the sub national geographies

Reply:
> > @kilemensi yes it's supposed to follow `geo` levels but I updated the function that gets the release years to search for them per country instead, I modified https://github.com/OpenUpSA/wazimap/blob/eccb51b5f28365b825e8320a49227ad81f4cfc6b/wazimap/data/utils.py#L345
> 
> Wouldn't it be better to leave the default behaviour so we don't end up diverging too much from wazimap? Is there any upside to doing it this way?

@kilemensi I feel like it's a cleaner way of approaching the problem, and any approach we take, we'd still divert from Wazimap because [this](https://github.com/OpenUpSA/wazimap/blob/eccb51b5f28365b825e8320a49227ad81f4cfc6b/wazimap/data/utils.py#L358) would have to be changed or edited

Reply:
Yes, we have to change it @phillipahereza but the question is how. This way means we can't really now say this release is only available at country level and not level 1 since you're now using just country names.

Where as if we try and maintain the Wazimap structure, we'll still be able to define that this release is application to Uganda but at country level and not level 1. Wazimap uses this structure in a lot of places, even in [Geo](https://github.com/OpenUpSA/wazimap/blob/eccb51b5f28365b825e8320a49227ad81f4cfc6b/wazimap/geo.py#L316) itself so maintaining it seems like a logic choice.

Reply:
> Yes, we have to change it @phillipahereza but the question is how. This way means we can't really now say this release is only available at country level and not level 1 since you're now using just country names.
> 
> Where as if we try and maintain the Wazimap structure, we'll still be able to define that this release is applicable to Uganda but at country level and not level 1. Wazimap uses this structure in a lot of places, even in [Geo](https://github.com/OpenUpSA/wazimap/blob/eccb51b5f28365b825e8320a49227ad81f4cfc6b/wazimap/geo.py#L316) itself so maintaining it seems like a logic choice.

ohh yeah, makes sense, but now how do we cleanly implement this `HURUMAP['tz']['available_release_years'] = {`

Reply:
Good question, which I'm totally happy to leave in you capable hands :sunglasses: @phillipahereza